### ISSUE TITLE
Question - Why is there no image normalization?

### ISSUE THREAD

**Issue by aniketmaurya** [keras-team/keras-io#383]
https://github.com/keras-team/keras-io/blob/b3361571f7cf13cfabdb1180dc9d47c83ffe4cb8/examples/vision/oxford_pets_image_segmentation.py#L76

Hi @fchollet, in the image segmentation example model is trained without input image normalization. It would be really helpful if you can mention the reason for that.

**Comment by sachinprasadhs** [comment]
Hi,

Thanks for reporting an issue.

Since the model architecture uses the batch normalization layer at the starting of the model, it eliminates or avoids the need of explicit normalization of the input input using `Min-Max scaling` or `Standardization` for this case.
Batch normalization layer adapts to handle data with different scales or ranges.

```
    # Entry block
    x = layers.Conv2D(32, 3, strides=2, padding="same")(inputs)
    x = layers.BatchNormalization()(x)
```
